lib/round.rb

```diff
@@ -0,0 +1,77 @@
+class Round
+
+  attr_reader :turns,
+              :deck,
+              :number_correct
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+  end
+
+  def current_card
+    deck.cards.first
+  end
+
+  def take_turn(guess)
+    turn = Turn.new(guess, current_card)
+    turns << turn
+
+    if turn.correct?
+      @number_correct += 1
+    end
+
+    deck.cards.rotate!
+    turn
+  end
+
+  def number_correct_by_category(category)
+    correct_by_category = 0
+    turns.each do |turn|
+      if turn.card.category == category
+        if turn.correct?
+          correct_by_category += 1
+        end
+      end
+    end
+    correct_by_category
+  end
+
+  def percent_correct
+    (@number_correct / turns.length.to_f) * 100.0
+  end
+
+  def percent_correct_by_category(category)
+    number_in_category = 0
+    turns.each do |turn|
+      if turn.card.category == category
+        number_in_category += 1
+      end
+    end
```

Comment:
Other than incrementing `number_in_category` this portion of the method is very similar to `cards_in_category` method on `Deck`. You can reuse that method here since we assume we've gone through the whole deck of cards by the time we calculate percentage correct. That cleans up a little bit of redundancy in the code to get it DRYer.